packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -155,13 +155,19 @@ const isCampaignEvent = (event: Event) => {
 
 export const shouldTrackHistoryPageView = (
   trackingOption: Options['trackHistoryChanges'],
-  newURL: string,
-  oldURL: string,
+  newURLStr: string,
+  oldURLStr: string,
 ): boolean => {
   switch (trackingOption) {
-    case 'pathOnly':
-      return newURL.split('?')[0] !== oldURL.split('?')[0];
+    case 'pathOnly': {
+      if (oldURLStr == '') return true;
```

Comment:
`previousURL` is null at first https://github.com/amplitude/Amplitude-TypeScript/blob/812713c51b0b69909897d8ee0b1d72032266e2a1/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L46C69-L46C86. Could you also update the test?
```suggestion
      if (oldURLStr == null) return true;
```

Reply:
We are passing un empty string while previousURL is null.

` shouldTrackHistoryPageView(trackHistoryChanges, newURL, previousURL || '') && shouldTrackOnPageLoad();`

https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L51